src/main/java/org/irods/Serialize/ModifyPermissionsOperations.java

```diff
@@ -16,9 +18,19 @@ public class ModifyPermissionsOperations {
     @JsonProperty("acl")
     private String acl;
 
-    public ModifyPermissionsOperations(String entity_name, String acl) {
+    public ModifyPermissionsOperations(String entity_name, Permission acl) throws IrodsException {
         this.entity_name = entity_name;
-        this.acl = acl;
+        switch (acl) {
+            case NULL:
+            case READ:
+            case WRITE:
+            case OWN:
+                this.acl = acl.getValue();
+                break;
+            default:
+                throw new IrodsException("'" + acl.getValue() +
+                        "' is not a supported Permission in modifyPermission operation.");
```

Comment:
Let's tell the dev what the value was expected to be. For example:
```
Expected one of the following Permission enum values: OWN, WRITE, READ, NULL
```

Reply:
Done.